cmdutil/metrics/otel/config.go

```diff
@@ -9,8 +9,9 @@ import (
 // Config is a reusable configuration struct that contains the necessary
 // environment variables to setup an metrics.Provider
 type Config struct {
-	Enabled             bool     `env:"ENABLE_OTEL_COLLECTION"`
-	CollectorURL        *url.URL `env:"OTEL_COLLECTOR_URL"`
 	MetricsDestinations []string `env:"OTEL_METRICS_DESTINATIONS,default=honeycomb;argus"`
 	Honeycomb           honeycomb.Config
+	CollectorURL        *url.URL `env:"OTEL_COLLECTOR_URL"`
+	Enabled             bool     `env:"ENABLE_OTEL_COLLECTION"`
+	UseExactAggregator  bool     `env:"OTEL_USE_EXACT_AGGREGATOR"`
```

Comment:
There are probably several options for aggregation. If we have to utilize a different one in another case we would end up with competing boolean config options. My proposal would be to map the config to an aggregator which allows you to leave it unset with the default option:

```
Aggregator string `env:"OTEL_AGGREGATOR"`
```

Reply:
That is how I tested it elsewhere. 

The new default is a good default choice for most metrics exports (see: https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric/selector/simple#NewWithHistogramDistribution) which seems to make sense for a library this one.

The presence of the config var was mostly to allow folks to option of immediately turning the new behaviour on/off while addressing any changes that will occur with the exported metrics.

If anyone absolutely needs to override this default behaviour and use the other aggregator it is easy enough to do by using  `otel.WithAggregator(simple.NewWithInexpensiveDistribution())` to construct their Otel provider.